### ISSUE TITLE
Push not pushing

### ISSUE THREAD

**Issue by bigardone** [saschatimme/elm-phoenix#2]
Hi!
I've been using your package quite a lot, but today I'm having an issue I haven't had before. I'm trying to push a message through the socket, but it's actually not doing anything. By anything, I mean that no message by received in the channel and no error is returned. 
Here's the line where I'm having the issue:

https://github.com/bigardone/phoenix-trello/blob/elm-version/web/elm/src/Boards/Update.elm#L87

And here is a screenshot of Chrome's developer tools:

![image](https://cloud.githubusercontent.com/assets/1090272/19625561/4fd8f864-991c-11e6-9985-179c487061c4.png)

So when submitting the form, it tries to send the message, but nothing happens. What might be going wrong? Thanks in advance!

**Comment by saschatimme** [comment]
Hey,

I took a look and your push message should be completely fine. One thing I stumbled about was
https://github.com/bigardone/phoenix-trello/blob/elm-version/web/elm/src/Subscriptions.elm#L26. It would be better to use

``` elm
Socket.init "ws://localhost:4000/socket/websocket"
    |> Socket.withParams [("token", jwt)]
```

(This eliminates the need to pass the jwt around. Also I use internally the endpoint url as a dict key, so it should not change).
Additionally you could enable some debug messages with `Socket.withDebug`. Also I pushed a minor change, such that with the debug flag reasons are printed why a message wasn't pushed.

P.S. I tried to get the elm-version of your project running, but I got stuck at the sign up (it currently doesn't create a new account)

**Comment by bigardone** [comment]
Thank you so much for the tips and update! After replacing the package with the latest version and deleting the `elm-stuff/build-artifacts/` folder, it finally notified me that I was trying to push to a wrong channel id :) now it's working like a charm 🎉 
Also, the `Socket.withParams` is much more convenient so I have refactored that part as well.
Sorry for the sign-up issue, it's not implemented yet. I added a default user in the seeds file and I completely forgot about refactoring to Elm that part :)
Thanks again for your time and for this awesome library.
